addons/account/models/account_bank_statement_line.py

```diff
@@ -45,6 +45,7 @@ def default_get(self, fields_list):
         comodel_name='account.move',
         auto_join=True,
         string='Journal Entry', required=True, readonly=True, ondelete='cascade',
+        index='btree_not_null',
```

Comment:
A `required=True` field cannot contains NULLs, it should be a simple `index=True`.

Reply:
Fixed